uat/testing-features/src/main/resources/greengrass/features/ggmq-1.feature

```diff
@@ -0,0 +1,32 @@
+@GGAD
+Feature: GGMQ-1
+
+  As a developer, I can configure my client agents to connect and use MQTT
+
+  Background:
+    Given my device is registered as a Thing
+    And my device is running Greengrass
+    And my MQTT Client Control is running
+
+  Scenario: GGMQ-1-T1: As a customer, I can connect, subscribe and publish using client application to MQTT topic
+    When I create a Greengrass deployment with components
+      | aws.greengrass.clientdevices.Auth        | LATEST                                                        |
+      | aws.greengrass.clientdevices.mqtt.EMQX   | LATEST                                                        |
+      | aws.greengrass.clientdevices.IPDetector  | LATEST                                                        |
+      | aws.greengrass.client.Mqtt5JavaSdkClient | classpath:/greengrass/components/recipes/client_java_sdk.yaml |
+    Then the Greengrass deployment is COMPLETED on the device after 300 seconds
+    And agent "agent1" is connected to MQTT Client Control
+    When I create client device "test" with the following policy
```

Comment:
this implies an IoT Policy to me, but it isn't clear. Make it clear what this policy is exactly.

Reply:
Fixed b6f305abbeb79765805f08540493ea08711e51e5